client/src/pages/Contest/ContestForm/useStyles.ts

```diff
@@ -0,0 +1,83 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+  },
+  contestTitle: {
+    fontSize: 26,
+    textAlign: 'center',
+    margin: 30,
+    color: '#000000',
+    fontWeight: 700,
+    fontFamily: "'Poppins'",
```

Comment:
can we include Poppins as part of theme so every time you use Typography, it'll get automatically applied across the app

Reply:
issue is fixed now poppins was not needed here in the first place